Azure/communication-ui-library-ios
# [Bugfix] Hide setup control bar while joining call when video is on

## Purpose

* When video is on, and join call button is tapped waiting to join a call, we should hide the setup control bar
* When video is off, we don't need to hide that bar

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test


* Launch the app and tap on Start Experience
* Turn on video
* Tap on Join call button
* Check that `Video on`, `Mic off` and `Speaker` buttons are all hidden

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |
<img src="https://user-images.githubusercontent.com/107075081/203452235-3008e784-2938-499e-8763-881ba10bb44f.PNG" width=200 height=400> | <img src="https://user-images.githubusercontent.com/107075081/203452334-7200b22f-94f1-4f36-9c70-c9f4f41512b4.PNG" width=200 height=400>

When video is off, it behaves as before
<img src="https://user-images.githubusercontent.com/107075081/203452432-162ce9e5-8125-4834-850d-f3ff21604530.PNG" width=200 height=400>


## Other Information
